client-src/elements/chromedash-guide-new-page.js

```diff
@@ -132,7 +132,7 @@ export class ChromedashGuideNewPage extends LitElement {
 
   renderForm() {
     const newFeatureInitialValues = {owner: this.userEmail};
-    this.fieldValues.allFields = newFeatureInitialValues;
+    this.fieldValues.feature = this.feature;
```

Comment:
Thanks for doing this.  This code seems cleaner than what was there before.

Reply:
Right.  The previous code was a result of my multiple confusions about what was going on.  Now that I see that again, the feature object should be attached to all the fieldValues arrays.  so I wonder if we could just pass the fieldValues object to the check function, rather than the getFieldValue function, which can't do the right thing on its own.